### ISSUE TITLE
[BUG] - Offscreen element being reported onscreen in Edge WebView2

### ISSUE THREAD

**Issue by karpinsn** [microsoft/accessibility-insights-windows#1299]
**Describe the bug**
Accessibility Insights is flagging that an onscreen element should have its IsOffScreen property to false, yet the element in question is scrolled offscreen.

**To Reproduce**
1) Open Visual Studio with C# application
2) Start debugging,
3) Navigate to the "Diagnostic Tools" window
4) Make the window small so that the CPU Usage graph is scrolled offscreen
5) Run Accessibility Insights automated checks

**Expected behavior**
As the "CPU" button is offscreen (chevron next to title) it should have the IsOffScreen property set to true and not be reported as an issue

**Actual behavior**
Accessibility insights is reporting that the IsOffScreen property should be set to false due to the element being on screen

**Screenshots or .GIF**
Reported failure
![image](https://user-images.githubusercontent.com/464580/157489596-795b0fef-a899-45b6-b42b-ed9c482c8776.png)

Actual element after scrolling down
![image](https://user-images.githubusercontent.com/464580/157489700-51e0c3e4-5407-4a6b-9bc7-c98bf3049073.png)

**Desktop (please complete the following information):**
 - OS: Windows 11 21H2 (22000.556) 
 - Accessibility Insights for Windows Version: 1.1.1741.1
 - Target Application: Visual Studio hosting EdgeWebView2
 - Target Application Version: Visual Studio 17.1 - EdgeWebView2 Edge97

**Comment by madalynrose** [comment]
Hello, @karpinsn!

The part of this error message that is most relevant to you is the second half: "An element's IsOffScreenproperty must be false **_when its clickable point is on screen_**."

In the Details pane, you can select "include all properties that have values" to see the ClickablePoint. 

![details pane with clickable point showing next to highlighted offscreen area](https://user-images.githubusercontent.com/3230904/157735262-466b8bea-a601-43cc-a7df-963d5867e454.png)

One example of how this accessibility issue could manifest is for dictation users: if a control's clickable point is detected by dictation assistive technology it will surface that to the user. However, if the control is offscreen and scrolled under other UI, the UI on top would receive the click.

You can see a more in-depth explanation of the bug in [this comment](https://github.com/microsoft/axe-windows/issues/539#issuecomment-797633889) from a similar earlier issue, which was determined to be a framework-level UI bug.

**Comment by karpinsn** [comment]
Thanks for taking a look @madalynrose. This still seems odd because the clickablePoint is outside the bounding box of the entire document body. While yes, this point is technically still within the physical screen, the point is scrolled beyond the document body so it is not clickable and offscreen as far as the web browser control is concerned.

**Comment by lisli1** [comment]
Hi @karpinsn, it is a bit strange, but while the point is not clickable by a mouse, the main issue is that it can still be reached by assistive technology. The rule says that you should change the clickable point to reflect that the control is offscreen.
